.eslintrc.js

```diff
@@ -66,9 +69,6 @@ module.exports = {
     // This rule tells people to do something (import foo = require('foo')) which doesn't work
     // with babel compiled typescript.
     '@typescript-eslint/no-var-requires': 'off',
-    // Enforce template strings for styles over objects for consistent codebase
-    // We can switch this to object at anytime, but lowering to warn as we have a mixed condebase right now.
-    'emotion/syntax-preference': [1, 'string'],
```

Comment:
This whole file is prettier except the removal of this line, which we simply ignore everywhere.